.github/workflows/ci-linux.yml

```diff
@@ -56,7 +56,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: Install
         run: |
-          sudo apt-get install -y cmake ${{ matrix.cxx }}
+          sudo apt-get install -y cmake libboost-locale-dev libboost-dev ${{ matrix.cxx }}
```

Comment:
Boost is only required if CMake configuration phase is specified with
https://github.com/nanodbc/nanodbc/blob/5b52ff0c8fd70572ed610bab8104fd5be08d954e/CMakeLists.txt#L19

Reply:
Yep, looks like I defaulted to using Boost.Locale if Unicode support was built in. Fixed